docs/modules/getting-started/pages/get-started-docker.adoc

```diff
@@ -313,28 +313,69 @@ go get github.com/hazelcast/hazelcast-go-client
 +
 [source,go]
 ----
-import "github.com/hazelcast/hazelcast-go-client"
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+func main() {
+	ShutdownProperly := func(ctx context.Context, hz *hazelcast.Client, err error) {
+		errSd := hz.Shutdown(ctx)
+		switch {
+		case err != nil && errSd == nil:
+			log.Fatal(fmt.Errorf("%v", err))
+		case err != nil && errSd != nil:
+			log.Fatal(fmt.Errorf("%v; %v", err, errSd))
+		}
+	}
 
-func mapSampleRun() {
-  cb := hazelcast.NewConfigBuilder()
+	cb := hazelcast.NewConfig()
+	cb.ClientName = "hello-world" <1>
```

Comment:
I am not sure if naming client is part of the steps, but if it is not I suggest we stick to default configuration with `StartNewClient` API. It is simpler IMO. What do you think?

Reply:
It was the part of the example.

Reply:
Okay then, ignore this please 🙏